tests/python/test_graph.py

```diff
@@ -9,7 +9,7 @@
 
 supported_floating_types = [ti.f32] if platform.system() == "Darwin" else [ti.f32, ti.f64]
 
-supported_archs_cgraph = [ti.vulkan, ti.opengl]
+supported_archs_cgraph = [ti.vulkan]
```

Comment:
hmmm did you remove ti.opengl for unsupported data types?

Reply:
Done.